CHANGELOG.md

```diff
@@ -4,6 +4,26 @@ Changelogs for versions not listed here can be found at https://github.com/DataD
 
 ---
 
+## 2.21.1
+
+
+### Bug Fixes
+
+- ASM
+    - Ensure that no module from ASM are loaded when ASM is disabled or unavailable. SCA: This ensures that no module from IAST are loaded when IAST is disabled or unavailable.
+    - Resolve an issue where IAST modules could be loaded, even if disabled, which could create an ImportError exception on Windows.
+
+ - Code Security
+    - Runtime Code Analysis (IAST): Avoid imports of IAST native module when IAST is not enabled.
+
+- Profiling
+    - Fixes an issue where the profiler unnecessarily outputs log messages when it was unable to record a lock acquire event. 
+
+- Tracing
+    - aiohttp: Adds the environment variable `DD_AIOHTTP_CLIENT_DISABLE_STREAM_TIMING_FOR_MEM_LEAK` to address a potential memory leak in the aiohttp integration. When set to true, this flag may cause streamed response span timing to be inaccurate. The flag defaults to false.
+    - span: Fix issue where spans weren't being handled correctly and were not being sent when using a custom Exception class that raises an exception in <span class="title-ref">\_\_str\_\_</span>.
```

Comment:
```suggestion
    - span: Fix issue where spans weren't being handled correctly and were not being sent when using a custom Exception class that raises an exception in ``__str__``.
```
I prefer using raw quotes but personal preference so feel free to disregard this